.github/workflows/ci.yml

```
@@ -35,3 +35,10 @@ jobs:
       - run: pnpm install --frozen-lockfile
       - run: npx playwright install chromium
       - run: npm run e2e
+      - name: Upload test results
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v2
```

Let's see if this works:
```suggestion
      - uses: actions/upload-artifact@v2
        if: ${{ failure() }}
```